msteams-platform/bots/how-to/conversations/subscribe-to-conversation-events.md

```diff
@@ -399,7 +399,10 @@ async def on_teams_channel_restored(
 
 ### Team members added
 
-The `teamMemberAdded` event is sent to your bot the first time it is added to a conversation. The event is sent to your bot every time a new user is added to a team or group chat where your bot is installed. The user information that is ID, is unique for your bot and can be cached for future use by your service, such as sending a message to a specific user.
+The `teamMemberAdded` event is sent to your bot the first time it's added to a conversation. The event is sent to your bot every time a new user is added to a team or group chat where your bot is installed. After a team member is added, the bots receives unique channel ID as a conversation ID using the [conversationUpdate](#conversation-update-events) and [installationUpdate](#installation-update-event) events. You can also cache for future use, such as sending a message to a specific user.bots. The bot posts welcome message in the same channel by default instead.
```

Comment:
Why is `teamMemberAdded` formatted differently from "channel member added"? The actual activity is still `conversationUpdate`.

Reply:
@aosolis and @vule-msft, unable to find "channel member added".
As per the walkthrough installationUpdate and conversationUpdate, both are required for the activity: to receive welcome message in the same channel where a team member is added. Please elaborate the changes, if any.

Reply:
Sorry I meant "channel created". The formatting changes for "team members added" and "team members removed", which leads one to think that it's a different event. 

Compare:
![image](https://user-images.githubusercontent.com/767208/169956824-44e24025-c9d6-4378-a516-38b232212c06.png)